docs/REST API/rest-api.md

```diff
@@ -2009,7 +2012,9 @@ If `after` is being used with `created_at` or `updated_at`, the table must be sp
         "encryption_key_available": false,
         "enrollment_status": null,
         "name": "",
-        "server_url": null
+        "server_url": null,
+        "device_status": "unlocked",
+        "pending_action": ""
```

Comment:
@noahtalerman @mna @ghernandez345:
+ `device_status` can be: "unlocked" (default), "locked", "wiped". (Should this be `null` if device doesn't have MDM turned on? Or do you think defaulting to "unlocked" is OK for non-mdm hosts?) 
+ `pending_action` can be: "" (default), "lock", "unlock", "wipe" (so, the names of the actions as displayed in the UI, no "-ing")

@noahtalerman let me know if you see any potential issues with the naming of "device_status"; made the most sense in my mind

Reply:
@rachaelshaw 
> Should this be null if device doesn't have MDM turned on? 

Given that this is in the `"mdm"` object, I think it's ok to have it show as "unlocked"? If MDM is not turned on, I believe the whole "mdm" object will be omitted (I think?).

Reply:
hmm I may be wrong about that, maybe "mdm" is always present even if not enabled.